### PR TITLE
6.x: Enable Qt3d

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -91,7 +90,10 @@ requirements:
     - python
     - libxslt
     - libxml2
-    - qt6-main >=6.4.1
+    - qt6-main
+    - qt6-3d
+    - qt6-charts
+    - qt6-datavis3d
```
Comment (hmaarrfk) [inline recipe/meta.yaml:96]: can we keep these as optional dependencies?

Comment (hmaarrfk) [reply recipe/meta.yaml:96]: For webengine, i test a particular version of python for compatibility with the extensions.
https://github.com/conda-forge/pyside2-feedstock/blob/main/recipe/meta.yaml#L146

That allows us to test both with and without all the optional dependencies.

Comment (jschueller) [reply recipe/meta.yaml:101]: done

**Comment by hmaarrfk** [comment]
Thank you!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -146,17 +159,17 @@ test:
     - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
     - {{ cdt('xcb-util-wm-devel') }}          # [linux]
   imports:
-    #- PySide6.Qt3DAnimation
-    #- PySide6.Qt3DCore
-    #- PySide6.Qt3DExtras
-    #- PySide6.Qt3DInput
-    #- PySide6.Qt3DLogic
-    #- PySide6.Qt3DRender
+    - PySide6.Qt3DAnimation        # [py==310]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:162]: Seems like windows is failing to import the first Qt3D dependency. 

```
2022-12-13T16:40:13.9664919Z (%PREFIX%) %SRC_DIR%>"%PREFIX%\python.exe" -s "%SRC_DIR%\run_test.py" 
2022-12-13T16:40:14.0137497Z import: 'PySide6.Qt3DAnimation'
2022-12-13T16:40:14.1428625Z Traceback (most recent call last):
2022-12-13T16:40:14.1430423Z   File "D:\bld\pyside6_1670942492999\test_tmp\run_test.py", line 2, in <module>
2022-12-13T16:40:14.1431756Z     import PySide6.Qt3DAnimation
2022-12-13T16:40:14.1432893Z ImportError: DLL load failed while importing Qt3DAnimation: The specified module could not be found.
```

The test evironment has it installed
```
2022-12-13T16:37:16.5565271Z     pyside6:                   6.4.1-py310h2f681d6_1     local      
2022-12-13T16:37:16.5566216Z     python:                    3.10.8-h4de0772_0_cpython conda-forge
2022-12-13T16:37:16.5567504Z     python_abi:                3.10-3_cp310              conda-forge
2022-12-13T16:37:16.5568596Z     qt6-3d:                    6.4.1-hef61a30_3          conda-forge
2022-12-13T16:37:16.5569507Z     qt6-charts:                6.4.1-h63175ca_3          conda-forge
2022-12-13T16:37:16.5570544Z     qt6-datavis3d:             6.4.1-h63175ca_3          conda-forge
2022-12-13T16:37:16.5571620Z     qt6-main:                  6.4.1-h5679a45_5          conda-forge
2022-12-13T16:37:16.5572573Z     tk:                        8.6.12-h8ffe710_0         conda-forge
```

Comment (jschueller) [reply recipe/meta.yaml:162]: maybe some dependency is not exported

Comment (jschueller) [reply recipe/meta.yaml:162]: seems it not found at build:
`WARNING (pyside6,Lib/site-packages/PySide6/Qt3DAnimation.pyd): $RPATH/Qt63DAnimation.dll not found in packages, sysroot(s) nor the missing_dso_whitelist.`

Comment (jschueller) [reply recipe/meta.yaml:162]: oh yes, its the cloberring thingy I missed

Comment (jschueller) [reply recipe/meta.yaml:162]: https://github.com/conda-forge/qt-3d-feedstock/pull/5

Comment (hmaarrfk) [reply recipe/meta.yaml:162]: hmm, now data visualization seems to be found during the build process, but still can't get imported.

Comment (jschueller) [reply recipe/meta.yaml:162]: the qt-datavis3d build needed to be retriggered

**Pull request merged**